packages/zipkin-instrumentation-redis/src/zipkinClient.js

```diff
@@ -0,0 +1,54 @@
+const {Annotation} = require('zipkin');
+const redisCommands = require('redis-commands');
+module.exports = function zipkinClient(tracer, redis, options, serviceName = 'redis') {
+  function mkZipkinCallback(callback, id) {
+    return function zipkinCallback(...args) {
+      tracer.scoped(() => {
+        tracer.setId(id);
+        tracer.recordAnnotation(new Annotation.ClientRecv());
+      });
+      callback.apply(this, args);
+    };
+  }
+  function commonAnnotations(rpc) {
+    tracer.recordAnnotation(new Annotation.ClientSend());
+    tracer.recordServiceName(serviceName);
+    tracer.recordRpc(rpc);
+    tracer.recordBinary('redis.key', rpc);
```

Comment:
shouldn't this be `redis.command`? Also, since it is the span name (which you can search on), do we need to log it again here?